packages/jest-cli/src/run_jest.js

```diff
@@ -77,7 +77,7 @@ const processResults = (runResults, options) => {
       const filePath = path.resolve(process.cwd(), outputFile);
 
       fs.writeFileSync(filePath, JSON.stringify(formatTestResults(runResults)));
-      process.stdout.write(
+      options.outputStream.write(
```

Comment:
I only ended up changing it for the case when `--json --outputFile and --useStderr` are used at the same time. Changing it for `--json --useStderr` wihtout `--outputFIle` broke a lot of integration tests, which made me curious if that was even the expected behavior. That is why I left it the way it was.

Reply:
Makes sense this way. Does it still fix the issue?